### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD033.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#36] [General]
# https://blog.3place.cn/3place/fsd/FSD033.html

https://blog.3place.cn/3place/fsd/FSD033.html

**Comment by picklecai** [comment]
原来梅菲斯特变福耳库阿斯，是为了诱惑海伦。

**Comment by picklecai** [comment]
海伦

算了！我跟我丈夫一同乘船回来，
现在他打发我先行进入宫城；
他到底怀着什么鬼胎，我猜不出来。
我回来当他的妻子？还是当一位王后？
或者当牺牲，偿还王上的激烈的痛苦，
偿还希腊人长期忍受的不幸的灾难？
我被征服了；是不是俘虏，我不知道！

梅菲斯特的成功从海伦的怀疑开始。

**Comment by picklecai** [comment]
姊妹们，是怎么回事？我们的王后不是
迈着急促的步伐又向我们走回？
怎么回事，伟大的王后，你怎么会
在宫殿里面没有受到仆婢的欢迎，
却反而受到惊吓？请你不要隐瞒；
因为我看到你的脸上很不高兴〔481〕，
一种高贵的愤怒正在跟惊骇斗争。

梅菲斯特变丑而不是常貌或者美貌，也许是丑更能骇人。
海伦的怀疑是成功的第一步，这里海伦的惊吓是成功的第二步。

**Comment by picklecai** [comment]
海伦
别将我纷乱的心再搅得胡里胡涂。
就是现在，我也不知道我是哪一个。

揭开海伦不愿意回忆的过去，是梅菲斯特成功的第三步。
不堪、羞愧……暗示了“美貌并非正义”，（从丈夫的视角看）错了还是错了。

**Comment by picklecai** [comment]
福耳库阿斯

王后，他要拿你做牺牲！

明确说出海伦的怀疑，是梅菲斯特成功的第四步。
有了前面的铺垫，特别是海伦已经心乱了，这里的明确说出（虽然本质上仍然是未被证实的，并没有证据），容易被当成事实。

**Comment by picklecai** [comment]
福耳库阿斯
说来很简单：若要保全她自己，还要
加上你们，完全取决于王后一人。
必须要下定决心，而且要从速决定。

梅菲斯特成功的第五步：营造出海伦可以自己掌控自己命运的错觉。

**Comment by picklecai** [comment]
合唱
请问有没有跳舞的男子？

福耳库阿斯
头等货！头发金黄、精神饱满的年轻人。
洋溢着青春的香气。只有从前帕里斯
走近王后时才散发这种香气。

海伦
你已经
完全忘其所以，请说出最后的决定！

福耳库阿斯
这要由你说，你认真、明确地说一声“好！”
我就领你们前去那座城堡。

梅菲斯特成功的第六步：让海伦周围的人被利诱（色诱），让她们去劝说海伦答应。这是从对方内部瓦解。

**Comment by picklecai** [comment]
福耳库阿斯
为了那家伙，他也要对你如法炮制。
美人不能分享；完全独占的人
痛恨跟他人公有，情愿把美人弄死。

（远处传来喇叭声；合唱队员们心惊胆战。）

就像这种尖锐的喇叭之声要震破
人们的耳鼓和脏腑，嫉妒的利爪也同样
抓紧男子的胸心，他决不忘记他曾经
占有过、而现在失去、不再占有的东西。

梅菲斯特成功的第七步，也是最后一步：讲恐怖故事。让海伦回忆丈夫对情敌的做法，再迁移到自己身上来。一边是色诱，是生路。一边是恐怖，是死路。这是临门一脚。
这一步之后，终于成功。

**Reply by picklecai** [reply]
而且迫在眉睫，喇叭声起了很大的推动作用。deadline就在眼前，必须从速决断。这种情况下，诱惑就成功了。

**Comment by picklecai** [comment]
和王婆对潘金莲一样，如果没有潘金莲自己的不如意在先，没有海伦自己的怀疑在先，后面种种，都是无源之水、沙子上的建筑。

王婆的十步，和梅菲斯特的七步，都是魔鬼的罗网，很难逃脱。但魔鬼只是催化剂，本人心中的想法，才是根。
